src/modules.md

```diff
@@ -20,4 +20,4 @@ The project includes these modules:
   - __Hadoop Distributed File System (HDFS™)__: A distributed file system that provides high-throughput access to application data.
   - __Hadoop YARN__: A framework for job scheduling and cluster resource management.
   - __Hadoop MapReduce__: A YARN-based system for parallel processing of large data sets.
-  - __[Hadoop Ozone](https://hadoop.apache.org/ozone/)__: An object store for Hadoop.
+  - __[Hadoop Ozone](https://ozone.apache.org)__: An object store for Hadoop.
```

Comment:
we should move Ozone to related projects section?

Reply:
Thanks @jojochuang for issuing it.
IMHO +1, let me change the title of this Jira and move Ozone to related projects section